test/TTBasic/generatorBasicReplay.baseline

```diff
@@ -0,0 +1,3 @@
+gen.next() = {value: 2, done: false}
+

```

Comment:
Shouldn't this also print "gen.next() = {value: 3, done: false}"?

Reply:
yes.  I don't exactly understand why - was thinking it was because the state of that last `WScript.SetTimeout(...)` call was lost, but need to confirm

Reply:
this is fixed now.  THe issue was that we were calling `writeTTDLog` after last snapshot, and  when that happens, replay aborts.